source/agora/node/FullNode.d

```diff
@@ -255,7 +242,7 @@ public class Node : API
             this.network.gossipTransaction(tx);
 
             // then nominate
-            if (this.config.node.is_validator)
+            if (this.nominator !is null)
```

Comment:
This belongs in the Validator class. The purpose of splitting the classes was to not have to do these checks.

https://github.com/bpfkorea/agora/issues/586

In the definition of done, the 4th item in the list is: 

> Fields which only belong in the Validator class should not be present in the FullNode class. For example, the Nominator instance should only be visible in the Validator class.

So remove the `nominator` instance from the FullNode class.